go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/flanksource/canary-checker
 go 1.12
 
 require (
+	github.com/DATA-DOG/go-sqlmock v1.4.1
 	github.com/Microsoft/go-winio v0.4.14 // indirect
+	github.com/apex/log v1.1.2
```

Comment:
this module should be unnecessary

Reply:
yes, fixed with 58d62537a5403a106a72aa16c9a320079c946b0b